### ISSUE TITLE
Looking for Help trying to setup 2 panels as 1 chain from 2 different parallel soruces

### ISSUE THREAD

**Issue by Krystm** [hzeller/rpi-rgb-led-matrix#1707]
So I am using RPI2DMD clock and I am messing around with the config and I see they are using looks like rpi-rgb-led-matrix to power it. I have built 3 different clocks and a few other projects around the led-matrixes and this is the first time I have run into this. I have researched and I thought maybe I had found something that worked but in the end didn't seem to work correctly.

The issue I am having. I have 2 panels that are 128x64 P2.5 Panels w/DP5125D chip. I am using the electrodragon interface card. The panels work great, I tested them independently and did find I needed the Line E mod which I did and each panel individually works great no problem. Where the issue comes from is for some reason these panels don't have an Out Hub75 connector soldered on it. the pads are there but I thought No problem I will just use the 2nd output of the interface card. And now I can't seem to figure out the setting to make it work.

These are the things I have tried to make it work, If I make my settings 128x64 with 2 for X panels and 1 for Y  I only get output from the right (1st)  panel.  If I try 128x64 with 1X and 2Y I get the entire image squished into a vertical 128 x 128 scenario. At this point the 2nd panel does kick on as there is a clock that runs between gifs and that clock is split across the bottom and top of the 2 panels as if it were vertical.  

[clock]
[clock]

So then I tried making my config 256x64 as that is what Ideally I am trying for..  and this is where it gets weird.  The Gifs  are formatted to be 128x32 natively when they run take up the entire half the screen as the resolution for them (128x32) get doubled because it is the proper aspect ratio and this is exactly what I am going for. However, when the clock comes on it is now centered and split across the middle horizontally. So the right panel sees the top right half the clock and the left panel (assuming as it would be considered vertical) is seeing the bottom right half of the clock positioned at the top left of the panel.   

[  extra undef 128px    ][clock]
[  extra undef 128px    ][clock]

if I do 128x64 

So what I am guessing is regardless how I set it up the 2nd output is always under the next panel. So is there a command / option or way that I can essentially remap / relocate the panel so that instead of the 2nd parallel panel being below it would be essentially next in the first chain? I guess would be the best way to say it. I thought maybe U Mapper or V Mapper but couldn't quite figure it out. I did order new panels that have the Output port and these panels do have the solder points which I tested and do seem to work like the regular configs but not super comfortable trying to solder a header on to the pane with minimum space with the frame and everything.

I am still looking and have tried different configs but the panels do seem to work natively no problem as is no multi-plexing or led-row type. 

So Is it possible to take these 2 panels:
[128x64] parallel 0
[128x64] parallel 1 

and turn them into 1 chain that is 256x64px?
[parallel 1 panel][parallel 0 panel]

TL:DR - I Want to make 1 large horizontal 256x64px image with 2 panels (128x64) that are on their own parallel chain since they do not have a HUB75 connector output to easily chain them together.

Thanks! Absolutely love this project I use these things for everything lol.

**Comment by Krystm** [comment]
AND I got it! V-Mapper did it! But Reversed the left and Right side. Not a problem since I havent built anything yet but I will leave this here so if anyone runs across it will know to try the V-Mapper Option!

Thanks again!
